src/battle_script_commands.c

```diff
@@ -17236,3 +17248,76 @@ void BS_TryRevivalBlessing(void)
         MarkBattlerForControllerExec(gBattlerAttacker);
     }
 }
+
+void BS_CheckPokeFlute(void)
+{
+    NATIVE_ARGS();
+    u8 battler;
+    s32 i;
+    u32 monToCheck, status;
+    u16 species;
+    u8 abilityNum;
```

Comment:
```suggestion
    u16 abilityNum;
```
We have more than 255 abilities

Reply:
Oh, I wonder if other places where this isn't updated are broken as well.

Reply:
It should be updated elsewhere too, unless we missed any